File: Dmoj/Misc/Back To School '18: Harambe.py
s = input()
t = input()

k = int(input())

cnt = 0
flag = 1

for i in range(len(s)):
    cnt += (s[i] != t[i])

    if(s[i] == " " and t[i] != " "):
        flag = 0
    
    if(t[i] == " " and s[i] != " "):
        flag = 0


if(cnt > k or not flag):
    print("No plagiarism")
else:
    
    print("Plagiarized")

File: Dmoj/Misc/Back To School '18: Harambe.cpp
#include <bits/stdc++.h>
#include <iostream>

using namespace std;

int main(){
    string s; cin >> s;
    string t; cin >> t;
    cout << s << "\n";
    cout << t << "\n";

    int k; cin >> k;

    int cnt = 0;
    for(int i = 0; i < s.size(); i++) cnt += (s[i] != t[i] && s[i] != ' ' && t[i] != ' ');
    cout << (cnt <= k ? "Plagiarized" : "No plagiarism");


}